dts/bindings/tcpc/st,stm32-ucpd.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+description: ST STM32 family UCPD. The default values were taken
```

Comment:
Nitpick: "ST STM32 family USB Type C / Power Delivery"
Not every one is familiar with UCPD acronym (yet)

Reply:
done